client/src/pages/Dashboard.js

```diff
@@ -73,6 +125,34 @@ function Dashboard(props) {
           onClose={handleCreateDialogClose}
           onCreateClick={handleDialogCreateInterviewButtonClick}
         />
+        <OverallDialog
+          onClose={handleFeedbackDialogsClose}
+          onNextQuestionClick={handleOvearllNextQuestionClick}
+        />
+        <ReviewDialog
+          onClose={handleFeedbackDialogsClose}
+          onNextQuestionClick={handleReviewDialogNext}
+          onPreviousQuestionClick={handleReviewDialogPrevious}
+        />
+        <StrengthsDialog
+          onClose={handleFeedbackDialogsClose}
+          onNextQuestionClick={handleStrengthsDialogNext}
+          onPreviousQuestionClick={handleStrengthsDialogPrevious}
+        />
+        <WeaknessesDialog
+          onClose={handleFeedbackDialogsClose}
+          onNextQuestionClick={handleWeaknessesDialogNext}
+          onPreviousQuestionClick={handleWeaknessesDialogPrevious}
+        />
+        <RecommendationsDialog
+          onClose={handleFeedbackDialogsClose}
+          onNextQuestionClick={handleRecommendationDialogNext}
+          onPreviousQuestionClick={handleRecommendationDialogPrevious}
+        />
+        <AnythingElseDialog
+          onClose={handleFeedbackDialogsClose}
+          onSubmitClick={handleAnythingElseSubmitClick}
+        />
```

Comment:
Instead of selecting which dialog to open using a condition of the location / route in each of their files, you can wrap each of these dialogs in a React Route. See example below:

```
<Route path="/dashboard/:interviewId/feedback/6">
    <AnythingElseDialog
          onClose={handleFeedbackDialogsClose}
          onSubmitClick={handleAnythingElseSubmitClick}
        />
</Route>
```

You could also generalize the feedback dialog, so that it takes a questionID and then depending on teh question you would show the right question. However, i think that is too much right now. 

The reason you need the interview Id in the Route, is so you can fetch the feedback for that interview Id in that dialog and pre-populate it. That is for future consideration. 

To handle Next and Back buttons, you can actually just use react-router-dom elements like Link, etc. so that you can go to the right route when the Link button is used. If we generalized the Dialogs, we could even make those Links just go to QuestionId + 1 (future consideration only please).